addons/mail/static/src/models/channel_member_view.js

```diff
@@ -5,15 +5,37 @@ import { one } from '@mail/model/model_field';
 
 registerModel({
     name: 'ChannelMemberView',
-    identifyingFields: ['channelMemberListCategoryViewOwner', 'partner'],
+    identifyingFields: ['channelMemberListCategoryViewOwner', 'channelMember'],
+    recordMethods: {
+        /**
+         * Handles click on the avatar of the channel member in the member list
+         * of this channel.
+         */
+        onClickMemberAvatar() {
+            if (!this.channelMember.partner) {
+                return;
+            }
+            this.channelMember.partner.openChat();
+        },
+        /**
+         * Handles click on the name of the channel member in the member list of
+         * this channel.
+         */
+        onClickMemberName() {
+            if (!this.channelMember.partner) {
+                return;
+            }
+            this.channelMember.partner.openProfile();
+        },
+    },
     fields: {
         channelMemberListCategoryViewOwner: one('ChannelMemberListCategoryView', {
             inverse: 'channelMemberViews',
             readonly: true,
             required: true,
         }),
-        partner: one('Partner', {
-            inverse: 'channelMemberViews',
+        channelMember: one('ChannelMember', {
+            inverse: 'channelMemberView',
```

Comment:
bad inverse "rename" (even though it's actually a move), the inverse of a view should always be a many